packages/core/src/App/Components/Layout/Header/menu-link.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Text } from '@deriv/components';
+import { useIsRealAccountNeededForCashier } from '@deriv/hooks';
+import { isMobile, routes, getStaticUrl } from '@deriv/shared';
+import { isExternalLink } from '@deriv/utils';
+import { observer, useStore } from '@deriv/stores';
+import { localize } from '@deriv/translations';
+import { BinaryLink } from 'App/Components/Routes';
+
+type TMenuLink = {
+    data_testid: string;
+    icon: string;
+    is_active: boolean;
+    is_disabled: boolean;
+    is_hidden: boolean;
+    link_to: string;
+    onClickLink: () => void;
+    suffix_icon: string;
+    text: string;
+};
+
+const MenuLink = observer(
+    ({
+        data_testid,
+        icon = '',
+        is_active,
+        is_disabled,
+        is_hidden,
+        link_to = '',
+        onClickLink,
+        suffix_icon = '',
+        text,
+    }: Partial<TMenuLink>) => {
+        const { common, ui, client } = useStore();
+        const { has_any_real_account, is_virtual } = client;
+        const { setMobileLanguageMenuOpen } = common;
+        const { toggleReadyToDepositModal, toggleNeedRealAccountForCashierModal } = ui;
+        const real_account_needed_for_cashier = useIsRealAccountNeededForCashier();
+        const is_trade_text = text === localize('Trade');
```

Comment:
can we change the variable name?  is_trader_text looks like a boolean

Reply:
It is a boolean